src/tokens/types.ts

```diff
@@ -58,10 +58,7 @@ type TokenSignature = Signature | MAC;
 /**
  * Token signature in JSON
  */
-type TokenSignatureJSON = {
-  type: 'Buffer';
-  data: Array<number>;
-};
+type TokenSignatureJSON = string;
```

Comment:
If it is `JSON` suffix, it should be a JSON version of the signature. What is this now? Why is it string?

Reply:
I may have changed it to a binary string when fixing things, I can look into it. This would be related to https://github.com/MatrixAI/Polykey/pull/525#discussion_r1269074329

Reply:
Yea I don't think this is correct... changing the type here might have big impacts later down the line. I think this requires a review as to what exactly changed.

We can't change these types without considering global effects. You should review this and ensure that input/output types stay the same.

Until we have a proper issue that can examine any refactoring to be done here.

Reply:
This has been reverted. Turns out it wasn't needed.